docs/example-transactions.md

```diff
@@ -0,0 +1,372 @@
+# Example transactions
```

Comment:
See comment above. Let's reorg this page into `docs/specification/example-transactions.md` or even just `examples.md`.